Doc/reference/compound_stmts.rst

```diff
@@ -312,11 +319,17 @@ when leaving an exception handler::
    >>> print(sys.exc_info())
    (None, None, None)
 
+
 .. index::
    keyword: except_star
 
-The :keyword:`except*<except_star>` clause(s) are used for handling
-:exc:`ExceptionGroup`\ s. The exception type for matching is interpreted as in
+.. _except_star:
+
+:keyword:`!except*` clause
+--------------------------
+
+The :keyword:`except* <except_star>` clause(s) are used for handling
```

Comment:
```suggestion
The :keyword:`!except* <except_star>` clause(s) are used for handling
```

Reply:
Ah, thought I caught that one, sorry. I also found a number of others deeper in the various other re-organized subsections which I also fixed.